api/dependencies/file.py

```
@@ -282,160 +318,169 @@ def get_file_validator(max_size: int=None, allowed_types: List[str]=None,
             if analysis_report.validation_result == ValidationResult.MALICIOUS:
                 if self.enable_quarantine:
                     quarantine_path = quarantine_file(content, analysis_report)
-                raise HTTPException(status_code=status.
-                    HTTP_422_UNPROCESSABLE_ENTITY, detail=
-                    'File rejected - Malicious content detected')
+                raise HTTPException(
+                    status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+                    detail="File rejected - Malicious content detected",
+                )
             elif analysis_report.validation_result == ValidationResult.SUSPICIOUS and self.enable_quarantine:
                 quarantine_path = quarantine_file(content, analysis_report)
```

Pylint (3.3.8)

[refactor] Unnecessary "elif" after "raise", remove the leading "el" from "elif"

(R1720)